tests/core/padding-utils/test_padding.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import unicode_literals
```

Comment:
I hadn't noticed it in other files before, so can't tell for sure. If the travis checks pass without it I'd say remove it